elements/rh-cta/docs/00-overview.md

```diff
@@ -0,0 +1,228 @@
+
+Do not use a call to action link to trigger an action, like submitting a form or changing a state.
+
+{% alert title="Learn More" %}
+  Visit the [Button](https://ux.redhat.com/components/button/){target="_blank"}
+  or [Form](https://ux.redhat.com/components/form/){target="_blank"} pages to 
+  learn more about how to use buttons and forms.
+{% endalert %}
+
+{% example palette="lightest",
+           style="--inline-img-max-width: 800px;",
+           alt="Call to action component usage, buttons",
+           src="/elements/cta/cta-usage-buttons.svg" %}
+
+### Brick (icon)
+
+When grouping Brick (icon) variants, different icons can be used to communicate 
+different topics, but they all must be the same style, color, and height.
+
+{% example palette="lightest",
+           style="--inline-img-max-width: 684px;",
+           alt="Call to action component usage, Brick (icon) variant",
+           src="/elements/cta/cta-usage-brick-icon.svg" %}
+
+### Primary (white) {style="margin-block:var(--rh-space-3xl) 0"}
+
+The Primary (white) variant can be used if the red variant conflicts with other 
+elements or violates accessibility standards.
+
+{% endsection %}
+
+
+{% section %}
+## Best practices
+
+### Changing styles
+
+Do not modify the styles of any variant for any reason.
+
+{% example palette="wrong",
+           style="--inline-img-max-width:598px;",
+           alt="Call to action component, best practice 1",
+           src="/elements/cta/cta-bestpractice-1.svg" %}
+
+### Too many links
+
+Do not group more than three variants together otherwise a user might have trouble making a selection if there are too many link choices.
+
+{% example palette="wrong",
+           style="--inline-img-max-width:635px;",
+           alt="Call to action component, best practice 2",
+           src="/elements/cta/cta-bestpractice-2.svg" %}
+
+### Using multiple Primary variants
+
+Do not use multiple Primary variants in the same group, use of that variant is reserved to represent the most important link.
+
+{% example palette="wrong",
+           style="--inline-img-max-width:305px;",
+           alt="Call to action component, best practice 3",
+           src="/elements/cta/cta-bestpractice-3.svg" %}
+
+### Mixing variant groups
+
+Do not group more than two variants together anywhere.
+
+{% example palette="wrong",
+           style="--inline-img-max-width:405px;",
+           alt="Call to action component, best practice 4",
+           src="/elements/cta/cta-bestpractice-4.svg" %}
+
+### Mixing Bricks
+
+Do not group different Brick variants together anywhere, use one only.
+
+{% example palette="wrong",
+           style="--inline-img-max-width:552px;",
+           alt="Call to action component, best practice 5",
+           src="/elements/cta/cta-bestpractice-5.svg" %}
+
+### Changing hierarchy
+
+Do not change the hierarchy when grouping, variants that are higher in hierarchy should go first.
+
+{% alert state="warning", title="warning" %}
```

Comment:
- Alert heading is not inline with icon.
- Alter title `warning` should be sentence case (`Warning`)